### PR TITLE
To_lists of the 02nd of August

### PR DESCRIPTION
**Pull request by emmamarichal**
@chrissimpkins / @m4rc1e 

The Boutros authors changes (https://github.com/google/fonts/pull/7930) has been merged from 3 weeks, but is still in the dev server. 
When I run gen-push-list, it wasn't added to the lists automatically. The PR has the `designer profile` tag, but it should have `metadata/OFL` one. @m4rc1e, do you think this can cause the issue?

But now all is good, I added it manually.

### CODE REVIEW

File: to_production.txt
Comment (chrissimpkins) [inline to_production.txt:2]: Requires Workspace deny listing before we merge.

**Review by chrissimpkins — CHANGES_REQUESTED** [review]
Let's wait on the COLR family deny listing in WS to merge this.  I'll update here when we are ready to go.

**Comment by emmamarichal** [comment]
@chrissimpkins done! :)

**Review by chrissimpkins — APPROVED** [review]

**Comment by chrissimpkins** [comment]
Two of the CI checks are failing.  Otherwise good to go following the https://github.com/google/fonts/pull/7991/commits/1be413e84b38d04ab602692ac7598dbfb6c0c676 change.

**Pull request merged**
